resources/js/Pages/Collections/Components/CollectionOfTheMonth/VoteCountdown.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Button } from "@/Components/Buttons";
+
+interface TimeLeft {
+    days: number;
+    hours: number;
+    minutes: number;
+}
+
+const calculateTimeLeft = (): TimeLeft => {
```

Comment:
as discussed in slack, we'll now also have the `22h:12m:39s` variation that needs adding